content/general/concepts/sdlc/sdlc.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'SDLC'
+Description: 'The Software Development Life Cycle (SDLC) is a process that is used to efficiently produce cost-effective and high-quality software through a series of structured steps or phases.'
+Subjects:
+  - 'Mobile Development'
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Software Development Life Cycle (SDLC)** is a process that is used to efficiently produce cost-effective and high-quality software through a series of structured steps or phases. Essentially, it is a standard industry approach to software development that outlines an entire project and timeline.
+
+## Phases
+
+The SDLC phases are as follows:
+
+1. Requirement analysis is the planning phase of the cycle.
```

Comment:
Most of the bullet points contain the phrase "is the phase that" or something equivalent, which make points 3 & 4 stand out for missing that clause. Not particularly important, but it's something I noticed reading.

Reply:
I can standardize the first two points with the other points! I also noticed that the second phase should be the "defining" phase and the requirement gathering and analysis phase are terms to be associated with this first phase 👍